### ISSUE TITLE
Which kind of framework do you use?

### ISSUE THREAD

**Issue by John1231983** [deadpoppy/CheXpert-Challenge#1]
Could you tell me which kind of framework do you use? If it is possible, please provide the trained weight and train model file ASAP because I want to use it for one competition. Thanks

**Comment by yil8** [comment]
@John1231983 we use pytorch, the full open source is under progress

**Comment by ChaiBapchya** [comment]
Curious to know by when can we expect the code release?

**Comment by yil8** [comment]
@ChaiBapchya thank you so much for your interest in our work. Unfortunately, we are currently busy preparing MICCAI conference, which is happening on Oct/13-Oct/17. We will try to squeeze some time for the code release before that.

**Comment by hoangtuanvu** [comment]
@yil8 Do you train models on small dataset or big one? Which validation test on the competition?

**Comment by yil8** [comment]
@hoangtuanvu we train on big one resized to 1024. we use the default validation test on the competition

**Comment by hoangtuanvu** [comment]
@yil8 Thank you.

**Comment by gokriznastic** [comment]
@yil8 Can you tell what kind of models you use? Is it modified version of imagenet models like densenet or your novel model?

**Comment by yil8** [comment]
@gokriznastic yes, it's a combination of quite a few tricks, e.g. densenet + attention_refine, I'm asking our team members to open source it lol

**Comment by gokriznastic** [comment]
@yil8 ohh I see. that would be awesome! :)

**Comment by hoangtuanvu** [comment]
@yil8 I submitted my models using codalab. However, I did not see it on the leaderboard after 2 weeks. I also sent an email to Jirvin, but no response. Have you meet it yet?

**Comment by yil8** [comment]
@hoangtuanvu yeah...they are sometimes pretty slow...
